docs/src/Guides/23 Localisation.md

```diff
@@ -0,0 +1,40 @@
+# Localising
+
+So your bot has grown, and now you need to ~~localize~~ localise your bot. Well thank god we support localisation then, huh?
+
+To clarify; localisation is a feature of application commands that discord offers,
+this means the same command will have different names and descriptions depending on the user's locale settings.
+
+# How its made:
+
+Let's take this nice and simple `hello` command
+```python
+import dis_snek
+
+@dis_snek.slash_command(name="hello")
+async def hello_cmd(ctx: dis_snek.InteractionContext):
+    await ctx.send(f"Hello {ctx.author.display_name}")
+```
+This command was immensely popular, and now we have some 🇫🇷 French users. Wouldn't it be nice if we could speak their language.
+```python
+import dis_snek
+from dis_snek import LocalisedName
+
+@dis_snek.slash_command(name=LocalisedName(english_us="hello", french="salut"))
+async def hello_cmd(ctx: dis_snek.InteractionContext):
+    await ctx.send(f"Hello {ctx.author.display_name}")
+```
+All we need to do is set the field to a `Localised` object, and dis-snek and discord wil handle the rest for you.
+For extra flavour lets make this command more dynamic.
+```python
+import dis_snek
+from dis_snek import LocalisedName
+
+@dis_snek.slash_command(name=LocalisedName(english_us="hello", french="salut"))
+async def hello_cmd(ctx: dis_snek.InteractionContext):
+    await ctx.send(f"{ctx.invoked_name} {ctx.author.display_name}")
+```
+Simply by changing `"hello"` to `ctx.invoked_name` the command will always use whatever the user typed to greet them.
```

Comment:
Is there any way to get the locale from the context, rather than having to cross reference command names?

Reply:
`ctx.locale`

Reply:
That should definitely be in the guide

Reply:
Added in 20ffff9eeb699969bc04c2a749546a1138f05e29